postProcess/Video3D.py

```
@@ -1,144 +1,385 @@
+
+    Raises:
+        subprocess.CalledProcessError: If the external command fails
+
+    Note:
+        Basilisk C utilities output data to stderr by convention
+    """
+    p = sp.Popen(command, stdout=sp.PIPE, stderr=sp.PIPE)
+    stdout, stderr = p.communicate()
+    if p.returncode != 0:
+        error_msg = stdout.decode("utf-8") if stdout else stderr.decode("utf-8")
+        raise subprocess.CalledProcessError(p.returncode, command[0], output=error_msg)
```

Ruff (0.11.9)

Undefined name `subprocess`

(F821)